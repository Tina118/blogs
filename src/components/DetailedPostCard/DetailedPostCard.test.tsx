import { render } from '@testing-library/react';
import DetailedPostcard from './index';

test('renders main App', () => {
  const fragment = render(<DetailedPostcard  id={1} name="test" email="test@gmail.com" body="testing" />);
  expect(fragment).toMatchSnapshot();
});
