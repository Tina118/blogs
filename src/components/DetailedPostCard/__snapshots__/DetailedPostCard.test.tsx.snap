// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders main App 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="d-flex justify-content-center align-items-center"
      >
        <div
          class="card mb-3"
          style="width: 30rem;"
        >
          <img
            alt="post"
            class="card-img-top"
            height="200rem"
            src="https://source.unsplash.com/random/200x200?sig=1"
            width="20rem"
          />
          <div
            class="card-body"
          >
            <h5
              class="card-title"
            >
              test
            </h5>
            <h6
              class="card-subtitle mb-2 text-muted"
            >
              test@gmail.com
            </h6>
            <p
              class="card-text"
            >
              testing
            </p>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="d-flex justify-content-center align-items-center"
    >
      <div
        class="card mb-3"
        style="width: 30rem;"
      >
        <img
          alt="post"
          class="card-img-top"
          height="200rem"
          src="https://source.unsplash.com/random/200x200?sig=1"
          width="20rem"
        />
        <div
          class="card-body"
        >
          <h5
            class="card-title"
          >
            test
          </h5>
          <h6
            class="card-subtitle mb-2 text-muted"
          >
            test@gmail.com
          </h6>
          <p
            class="card-text"
          >
            testing
          </p>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
